CacheService/docker_context/CacheHandler.py

```diff
@@ -224,4 +267,117 @@ async def getQuote(self, trans_num, user_id, stock_id):
             )
             return result, status
 
+    async def getBulkQuote(self, user_ids, stock_ids, transaction_nums):
+        results = []
+        while stock_ids:
+            user_id = user_ids.pop(0)
+            stock_id = stock_ids.pop(0)
+            transaction_num = transaction_nums.pop(0)
+
+            check = await self.RedisHandler.rExists(stock_id)
```

Comment:
This looks like a lot of the same logic that is in getQuote. Could we call that method instead?